site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -208,3 +208,76 @@ If you want to build your own image starting with our base image, the following
 ```
 docker pull fusionauth/fusionauth-app
 ```
+
+== Kickstart
+
+Using Docker with Kickstart is a powerful combination. Using these technologies together lets you:
+
+* Configure and share development environments
+* Create replicable bug reports 
+* Spin up auth instances with a well known, versioned set of data for continuous integration and testing
+
+All the normal limitations of Kickstart apply (the Kickstart will not run if the database has already been set up with an API key, for example).
+
+It's easy to get started with Kickstart, but you'll need to tweak your Docker Compose files a bit. Before you being, you'll need a valid `kickstart.json` file. Check out the link:/docs/v1/tech/installation-guide/kickstart/[Kickstart documentation] for more information on writing one.
+
+Once you have a valid `kickstart.json` file, create a subdirectory in the location of your `docker-compose.yml` file. It can be named anything; this documentation will use a directory called `kickstart`. Next, you'll mount this directory and set the `FUSIONAUTH_KICKSTART` variable in the `docker-compose.yml` file. 
```

Comment:
`FUSIONAUTH_KICKSTART` is now the deprecated name. 

`FUSIONAUTH_APP_KICKSTART_FILE` is the new name. 
https://fusionauth.io/docs/v1/tech/reference/configuration/

I think I recently updated this in the main kickstart doc as well.
https://fusionauth.io/docs/v1/tech/installation-guide/kickstart/

Reply:
Awesome, great!